cdds/cdds/convert/configure_workflow/__init__.py

```diff
@@ -113,7 +118,12 @@ def run_cdds_convert(arguments: ConvertArguments, request: Request) -> None:
     workflow_manager.checkout_convert_workflow()
     workflow_manager.update()
     workflow_manager.clean_workflow()
-    workflow_manager.run_workflow()
+
+    if arguments.no_submit:
+        print("Skipping running cylc vip. Templated workflow path:")
```

Comment:
Can we either change these print statements to logger messages or add a logger.info message to make it clear that the user has requested that the workflow is not submitted (I can imagine us digging through logs trying to work this out otherwise)